docs/chain-details/architecture.md

```diff
@@ -0,0 +1,28 @@
+
+# Architecture
+
+## Overview
+
+Building a blockchain is not just about software/hardware development. Instead, it is a combination of technological design, incentive mechanism, game theory and governance, which together nourish a robust system that also allows for continuous innovation. Our initially proposed architecture may hence undergo future revisions in response to changes in incentives, governance or other external requirements.
+
+Crypto.com Chain is open to the public to join, participate and scrutinise related transactions. We do not expect that, for example, mobile clients will be able to perform heavy-lifting tasks and have a reliable always-online network connection. For that reason, there are two different types of nodes are responsible for various duties:
+
+- Validator (Council Node), responsible for validating and committing new blocks to the blockchain; and
+- Community Node (Full node), responsible for fetching the blockchain data and serve it upon the client's request.
```

Comment:
does introducing special terminology help anything? just use "full node" and "validator"

Reply:
[Updated](https://github.com/crypto-com/chain-docs/pull/38/commits/89c4e43c18b989be0b84687afa780187dfe932ee#diff-14fb79233d19786b43ce2f4bcff6ba5bfdef6c775705f8f0437b2c8a52c0e002R9) to "full node" and "validator"